### DISCUSSION TITLE
Imposing uniqueness of user names across tenants

### DISCUSSION THREAD

**Discussion by liorbalmas** [abpframework/abp#7865] [General]
Hi 
Can uniqueness of user names be imposed across tenants without customizing identity module?
Thanks

**Comment by maliming** [comment]
hi @liorbalmas 

You need custom modules.

**Reply by liorbalmas** [reply]
Do you mean I need to customize the module or to add some other module?

**Reply by liorbalmas** [reply]
Can adding a unique index to the users table manually do as dirty workaround?

**Reply by maliming** [reply]
You can't, You need to use code to check if there is a username that already exists.

**Reply by liorbalmas** [reply]
thanks
